### ISSUE TITLE
systemd "pretty hostname" ?

### ISSUE THREAD

**Issue by jhimanka** [nojhan/liquidprompt#596]
Shell: bash
Operating system: ubuntu 19.10 
Liquid Prompt version (tag, commit):  1.11-3ubuntu1

Systemd has the concept of regular hostname and pretty hostname. Is there a way to use the pretty hostname in the prompt? It can be accessed with `hostnamectl --pretty` or by parsing /etc/machine-info

**Comment by Rycieos** [comment]
Note that 1.11 is 3.5 years old, and there are 29 commits since then.

I think the name of "pretty *hostname*" is a misnomer, since it's not a hostname at all, it's just a name.

To answer your question, it's not a supported feature, but you can customize the prompt yourself to add it. See https://github.com/nojhan/liquidprompt#rearranging-the-prompt

You could setup a prompt like this:
```
export LP_PS1=`echo -ne "[\${LP_USER}\$(hostnamectl --pretty):\${BLUE}\$(pwd)\${NO_COL}] \${LP_GIT} \\\$ "`
```
Or with whatever you want.

**Comment by jhimanka** [comment]
Yes, i did get it working with something like you suggested. I was just wondering that systemd becoming more pervasive there might be interest in enabling easy use of the pretty hostname. This is systemd terminology, and indeed it's not the actual hostname. 

I'm ok with closing the issue, I just wanted to point out that such a feature exists and might be useful. My laptop's hostname is eidolon and it's nice to see it in the prompt as εἴδωλον.

**Comment by Rycieos** [comment]
I think it's a valid feature request. I'd leave it open.
